src/main/java/CensusData.java

```diff
@@ -3,6 +3,7 @@
 import com.googlecode.objectify.annotation.Cache;
 import com.googlecode.objectify.annotation.Entity;
 import com.googlecode.objectify.annotation.Id;
+import java.util.List;
```

Comment:
Doesn't look like you're using this

Reply:
Good point. I was earlier but later deleted that functionality.